jquery.flot.js

```diff
@@ -2604,18 +2604,22 @@ Licensed under the MIT license.
             }
 
             if (!redrawTimeout) {
-                redrawTimeout = setTimeout(drawOverlay, t);
+                redrawTimeout = setTimeout(function() {
+                    drawOverlay(plot);
+                }, t);
             }
         }
 
-        function drawOverlay() {
+        function drawOverlay(plot) {
             redrawTimeout = null;
 
             if (!octx) {
                 return;
             }
             overlay.clear();
             executeHooks(hooks.drawOverlay, [octx, overlay]);
+            var event = new CustomEvent('onDrawingDone');
```

Comment:
Please, make sure the new event is mentioned in the documentation: https://github.com/ni-kismet/engineering-flot/blob/728b73fcd7411911877a1827aa31d271680e74f0/API.md#plot-methods in the **triggerRedrawOverlay** section.

Maybe here as well: https://github.com/ni-kismet/engineering-flot/blob/728b73fcd7411911877a1827aa31d271680e74f0/API.md#hooks in the **drawOverlay** section.

\+ specify this small API enhancement in the CHANGELOG.md file.